src/main/webapp/search-results.html

```diff
@@ -100,33 +100,27 @@
                 <h4 class="container-label" id="num-tutor-results"></h4>
                 <div id="tutors-container"></div>
             </div>
+
             <div id="books">
-                <div id = "tutor-books">
-                    <h4 class="container-label" id="tutor-books-label">Tutors' Recommendations</h4>
-                    <div id="lists-container" class="list-inline"></div>
-                    <!-- Modal -->
-                    <div class="modal fade" id="view-list-modal" tabindex="-1" role="dialog" aria-hidden="true">
-                        <div class="vertical-alignment-helper">
-                            <div class="modal-dialog modal-dialog-centered vertical-align-center" role="document">
-                                <div class="modal-content">
-                                <div class="modal-header">
-                                    <h3 class="modal-title" id="list-modal-name" style="display:inline-block"></h3>
-                                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-                                    <span aria-hidden="true">&times;</span>
-                                    </button>
-                                </div>
-                                <div class="modal-body" id="list-modal-body" ></div>
-                                <div class="modal-footer">
-                                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-                                </div>
-                                </div>
+                <div class="accordion" id="books-accordion">
+                    <div class="panel" id="tutor-books-panel">
+                        <button type="button" class="btn" data-toggle="collapse" data-target="#collapsible-1" data-parent="#books-accordion">Tutors' Recommendations</button>
+                        <div id="collapsible-1" class="collapse">
```

Comment:
What does collapse do?

Reply:
Basically, if you click the button, it will hide or show the information. There are more examples here: https://getbootstrap.com/docs/4.3/components/collapse/
I used the accordion example.